spaceship/lib/spaceship/portal/certificate.rb

```diff
@@ -303,6 +308,12 @@ def create!(csr: nil, bundle_id: nil)
           response['certificateTypeDisplayId'] = response['certificateType']['certificateTypeDisplayId']
           self.new(response)
         end
+
+        # Default portal class to use when finding by bundle_id
```

Comment:
maybe move this up so the definition is before the first usage of it?
(honestly don't know if this matters for the actual code, but I just wondered where this came from in line 297, wrote a comment, and then just found this here when scrolling down later... so having it up would have a benefit)